eng/build.ps1

```diff
@@ -612,6 +612,9 @@ function Deploy-VsixViaTool() {
   # Disable text spell checker to avoid spurious warnings in the error list
   &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Editor\EnableSpellChecker" Value dword 0
 
+  # Run source generators automatically during integration tests.
+  &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Roslyn\RunSourceGeneratorsWhenBuildsComplete" Value dword 0
```

Comment:
This should only be necessary if the option requires restarting the IDE after changing its value. Otherwise, the settings in StateResetInProcess would be sufficient.